src/lib/yang/schema.lua

```diff
@@ -700,10 +707,11 @@ function resolve(schema, features)
                   node.body[k] = v
                end
                -- TODO: Handle refine and augment statements.
-            else
-               node.body[k] = visit(v, env)
             end
          end
+         for k,v in pairs(node.body or {}) do
```

Comment:
This is quadratic, can you add a separate post-pass instead to run on the result of resolve(), and instead of propagating the config down in "env" here, propagate it down the tree in that pass.  I.e. a separate function that runs on the result of resolve() that just traverses down the ".body" of the nodes.